### ISSUE TITLE
"In English?" button is not reachable with keyboard

### ISSUE THREAD

**Issue by erinchambers** [CDRH/family_letters#203]
The button on the [Learn page](https://familyletters.unl.edu/en/learn) that says "In English?" is currently two spans, and since those aren't interactable elements, it's not in the tab order of the page and you can't get to it with a keyboard only.

This is an easy fix; just need to change the `span` with the background image into a semantic `<button>`. Could also tweak the CSS so the text has the same style change on hover as regular links.

**Comment by erinchambers** [comment]
Fixed in `a11y_erin` branch, PR coming soon

**Comment by erinchambers** [comment]
Realized the code for this button also had the page in Spanish reading "En español?" when it needed to say "En inglés?". Fix in PR #207.
